client/pkg/pynuoca.py

```diff
@@ -0,0 +1,62 @@
+# (C) Copyright NuoDB, Inc. 2020  All Rights Reserved.
+#
+# Add the nuodb collection agent
+
+import os
+
+from client.package import Package
+from client.stage import Stage
+from client.artifact import PyPIMetadata, Artifact
+from client.utils import *
```

Comment:
I'm trying to move away from using `import *`.  I've added to my Emacs a Python LSP client which incorporates various Python checks etc. and it doesn't like this; I'm trying to import the specific methods.